src/server/actions/indicator.ts

```
@@ -0,0 +1,74 @@
+      const olhcvList = await getDexOhlcv(topPoolId, 'solana', TIMEFRAME.MINUTES, '1');
+      let result;
+
+      const closingValues = getClosingValues(olhcvList , fromTimestamp);
+
+      if(closingValues.length === 0) {
+        return { success: false, error: 'No closing values found' };
+      }
+        
+      switch (indicator) {
+        case 'rsi':
+            const rsiResult = rsi(closingValues, { period: parameters.period || 14 });
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)